config.sh

```diff
@@ -27,6 +27,35 @@ BZIP2_VERSION=1.0.8
 LIBXCB_VERSION=1.15
 BROTLI_VERSION=1.0.9
 
+function fuse_macos_intel_arm64 {
+    local wheelhouse=$(abspath ${WHEEL_SDIR:-wheelhouse})
+    local py_osx_ver=$(echo ${MB_PYTHON_OSX_VER} | sed "s/\./_/g")
+    mkdir -p tmp_fused_wheelhouse
+    for whl in $wheelhouse/*.whl; do
+       if [[ "$whl" == *macosx_${py_osx_ver}_x86_64.whl ]]; then
+           whl_base=$(sed "s/macosx_${py_osx_ver}_x86_64.whl//" <<< $whl)
+           if [[ -f "${whl_base}macosx_11_0_arm64.whl" ]]; then
+               delocate-fuse $whl "${whl_base}macosx_11_0_arm64.whl" -w tmp_fused_wheelhouse
+               mv tmp_fused_wheelhouse/$(basename $whl) $wheelhouse/$(basename ${whl_base})macosx_${py_osx_ver}_universal2.whl
+               rm $whl
+           fi
+       fi
+    done
+}
```

Comment:
This change has been approved, but merged, in https://github.com/multi-build/multibuild/pull/495

Reply:
https://github.com/multi-build/multibuild/pull/495 has been merged now, so I've updated multibuild to include it, and removed this patch.